docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -6,10 +6,29 @@ Costing Method Parameters
 
 The following parameters are constructed for the unit on the FlowsheetCostingBlock (e.g., `m.fs.costing.ion_exchange`) when applying the `cost_ion_exchange` costing method in the ``watertap_costing_package``:
 
-.. csv-table::
-   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units", "Notes"
```

Comment:
If possible, we should replace instances of $ with USD_YYYY

Reply:
I also just noticed I have a "Notes" column here since I had it on the previous doc. Should I remove or consolidate with the Description column? It provides some more detail on the source of the costing data and/or how it can be used in the model

Reply:
> If possible, we should replace instances of $ with USD_YYYY

with the underscore?

Reply:
Preferably with the underscore

Reply:
I think it's fine to keep the Notes section as it is.